pyproject.toml

```diff
@@ -132,7 +135,38 @@ add-ignore = ['D107', 'D407', 'D417']
 
 [tool.setuptools]
 include-package-data = true
+license-files = ['LICENSE']
 
 [tool.setuptools.packages.find]
 include = ['copulas', 'copulas.*']
 namespaces = false
+
+[tool.bumpversion]
+current_version = "0.10.1.dev0"
+commit = true
+tag = true
+parse = '(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<release>[a-z]+)(?P<candidate>\d+))?'
+serialize = [
+	'{major}.{minor}.{patch}.{release}{candidate}',
+	'{major}.{minor}.{patch}'
+]
+
+[tool.bumpversion.parts.release]
+optional_value = 'release'
+values = [
+	'dev',
+	'release'
+]
+
+[tool.bumpversion.parts.candidate]
+
+[[tool.bumpversion.files]]
+filename = 'pyproject.toml'
+search = 'current_version = "{current_version}"'
+replace = 'current_version = "{new_version}"'
```

Comment:
I think this is done automatically by `bump-my-version`.

Reply:
good catch, thanks!